### PR TITLE
V2020.q3 final

### PR DESCRIPTION
**Pull request by huntermccallum**
Review of changes between Final and current Testing Branch

### CODE REVIEW

File: Configs/MissionSettings.md [hunk 1]
```diff
@@ -899,11 +899,11 @@ diwako_dui_enable_compass = true;
 diwako_dui_enable_compass_dir = 1;
 diwako_dui_enable_occlusion = false;
 diwako_dui_enable_occlusion_cone = 360;
-diwako_dui_hudScaling = 2;
+force diwako_dui_hudScaling = 1;
```
Comment (xNymia) [inline Configs/MissionSettings.md]: Is there a real need to force this scaling setting? would be a bit of a pain in the ass for people running higher resolutions

File: Configs/ServerSettings.md [hunk 2]
```diff
@@ -899,11 +899,11 @@ diwako_dui_enable_compass = true;
 diwako_dui_enable_compass_dir = 1;
 diwako_dui_enable_occlusion = false;
 diwako_dui_enable_occlusion_cone = 360;
-diwako_dui_hudScaling = 2;
+force diwako_dui_hudScaling = 1;
```
Comment (xNymia) [inline Configs/ServerSettings.md]: Is there a real need to force this scaling setting? would be a bit of a pain in the ass for people running higher resolutions

File: Configs/MissionSettings.md (hunk 1 above)
Comment (huntermccallum) [reply Configs/MissionSettings.md]: This was an export of Sniper's settings, so I think it was done by accident. I will remove the forced HUD scaling.

File: Configs/ServerSettings.md (hunk 2 above)
Comment (huntermccallum) [reply Configs/ServerSettings.md]: This was an export of Sniper's settings, so I think it was done by accident. I will remove the forced HUD scaling.

**Review by xNymia — APPROVED** [review]

File: Configs/ServerSettings.md (hunk 2 above)
Comment (xNymia) [reply Configs/ServerSettings.md]: 🚢

File: Configs/MissionSettings.md (hunk 1 above)
Comment (xNymia) [reply Configs/MissionSettings.md]: 🚢

**Pull request merged**